buildSrc/build.gradle.kts

```diff
@@ -5,4 +5,5 @@ plugins {
 dependencies {
    implementation(libs.testlogger.gradle.plugin)
    implementation(libs.kotlin.gradle.plugin)
+   implementation("io.kotest:kotest-framework-multiplatform-plugin-gradle:5.9.1")
```

Comment:
Why?

Reply:
Oops, I left this in my mistake. I've removed it.